cms/jinja2/templates/pages/statistical_article_page--previous-releases.html

```diff
@@ -1,28 +1,80 @@
 {% extends "templates/base_page.html" %}
 
+{% from "components/document-list/_macro.njk" import onsDocumentList %}
+{% from "components/hero/_macro.njk" import onsHero %}
+{% from "components/pagination/_macro.njk" import onsPagination %}
+
+{% block header_area %}
+    {%- set hero_topic = _("Previous Releases") -%}
+    {%- set hero_title = _("Previous Releases") -%} {# TODO: Needs to be updated to use related Statistical Article title #}
```

Comment:
the previous releases page title is the 
Series title + " Statistical articles" (extrapolated from the live site).

ref: https://www.ons.gov.uk/economy/inflationandpriceindices/bulletins/consumerpriceinflation/previousreleases
https://www.ons.gov.uk/economy/grossdomesticproductgdp/bulletins/gdpfirstquarterlyestimateuk/previousreleases

```suggestion
    {%- set hero_title = page.title ~ " " ~ _("Statistical articles") -%}
```